### PR TITLE
document ExprGraph

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

File: cmd/bosun/web/chart.go [hunk 1]
```diff
@@ -165,6 +165,8 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}, nil
 }
 
+//ExpGraph returns an svg graph.
+//the basename of requested svg file should be a base64 encoded expression
```
Comment (kylebrandt) [inline cmd/bosun/web/chart.go]: Spaces after comment chars, i.e. `// Foo boo baz.`
Capitalization of "The" in the second sentence and fix missing period.

**Comment by Dieterbe** [comment]
ohhh shittt. i have some work to do on this. stay tuned.

File: cmd/bosun/web/chart.go [hunk 2]
```diff
@@ -165,6 +165,8 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}, nil
 }
 
+// ExpGraph returns an svg graph.
```
Comment (kylebrandt) [inline cmd/bosun/web/chart.go]: ExprGraph not ExpGraph

**Comment by Dieterbe** [comment]
Wow I'm so good at this stuff wohoha . stay tuned

**Comment by Dieterbe** [comment]
how about now

File: cmd/bosun/web/chart.go [hunk 3]
```diff
@@ -165,6 +165,8 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}, nil
 }
 
+// ExprGraph returns an svg graph.
+// The basename of requested svg file should be a base64 encoded expression.
```
Comment (kylebrandt) [inline cmd/bosun/web/chart.go]: The basename of *the* requested svg file should be a base64 encoded expression.
Is it optional, if not?
"The basename of the requested svg file must be a base64 encoded expression.

Maybe I'm getting ridiculous? :-/

**Comment by Dieterbe** [comment]
whoa, i'm usually much better at writing stuff down. you can be pedantic, sorry for the back and forth :)

**Pull request merged**